NBA/server/services/nba_api.py

```diff
@@ -0,0 +1,33 @@
+import json
+import requests
+
+async def get_players(year,team):
+    teams = await fetch_data(year,'teams')
+    players = await fetch_data(year,'players')
+    team_id = get_team_id(teams,team)
+    team_players = [player for player in players if player["teamId"] == team_id]
```

Comment:
If no player has the team id? returning None?

Reply:
an empty list